src/annotator/index.js

```diff
@@ -33,34 +33,29 @@ const appLinkEl = /** @type {Element} */ (
   )
 );
 
-const config = configFrom(window);
+const annotatorConfig = getConfig('annotator');
 
 function init() {
   const isPDF = typeof window_.PDFViewerApplication !== 'undefined';
 
-  if (config.subFrameIdentifier) {
+  if (annotatorConfig.subFrameIdentifier) {
     // Other modules use this to detect if this
     // frame context belongs to hypothesis.
     // Needs to be a global property that's set.
     window_.__hypothesis_frame = true;
   }
 
   // Load the PDF anchoring/metadata integration.
-  config.documentType = isPDF ? 'pdf' : 'html';
+  annotatorConfig.documentType = isPDF ? 'pdf' : 'html';
 
   const eventBus = new EventBus();
-  const guest = new Guest(document.body, eventBus, config);
-  const sidebar = !config.subFrameIdentifier
-    ? new Sidebar(document.body, eventBus, guest, config)
+  const guest = new Guest(document.body, eventBus, annotatorConfig);
```

Comment:
There is an existing complication here that the `Guest` passes its `config` argument through to the `CrossFrame` constructor which in turn passes it through to the `injectHypothesis` utility that injects configuration into child frames. I think it should be OK to pass just the guest-related configuration to the `Guest` constructor because in child frames only the `Guest` part of the annotator is instantiated and not the `Sidebar` or `Notebook` parts.

Reply:
Understood and I think we need to strongly consider that for the audit which is why I want to continue to pass the total config to the guess for now even though it's called "annotatorConfig"